android-configure

```diff
@@ -16,5 +16,5 @@ export PLATFORM=android
 
 if [ $2 -a $2 == 'gyp' ]
   then
-    ./gyp_uv.py -Dtarget_arch=arm -DOS=android
+    ./gyp_uv.py -Dtarget_arch=arm -DOS=android -f make-android
```

Comment:
If we don't specify the target as "android", the gyp scripts don't handle android cross compilation properly and end up trying to compile against a darwin target environment and using the wrong include files, libs, etc.